docs/crd-changelog.md

```diff
@@ -0,0 +1,160 @@
+# CRD Changelog
+
+This document explains major changes made in new CRD versions. It is intended to help users migrate and take
+advantage of the new features.
+
+## OpenTelemetryCollector.opentelemetry.io/v1beta1 
+
+### Migration
+
+There is no need for any immediate user action. The operator will automatically convert existing `v1alpha1` resources to `v1beta1`.
+
+However, the conversion procedure will leave your resources stored as both versions. Support for `v1alpha1` will be removed in the future.
+In order to migrate fully to `v1beta1`:
+
+1. Update any manifests you have stored outside the cluster, for example in your infrastructure git repository.
+2. Apply them, so they're all stored as `v1beta1`.
+3. Update the OpenTelemetryCollector CRD to only store `v1beta1`
+   ```bash
+   kubectl patch customresourcedefinitions opentelemetrycollectors.opentelemetry.io  \
+     --subresource='status' \
+     --type='merge' \
+     -p '{"status":{"storedVersions":["v1beta1"]}}'
+   ```
+For a more thorough explanation of how and why this migration works, see the relevant [Kubernetes documentation][crd_migration guide].
```

Comment:
shouldn't there be a link for the kube documentation there?

Reply:
actually this may just be the github preview renderer not working

Reply:
no, nvm i think this should be []() syntax right?

Reply:
Yeah, it was missing an underscore.